userguide/storage.rst

```diff
@@ -1371,17 +1371,22 @@ changed, and destroying a zvol requires confirmation.
 Managing Encrypted Volumes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-%brand% generates and stores a randomized *encryption key* whenever
+%brand% generates a randomized *encryption key* whenever
 a new encrypted volume is created. This key is required to read and
-decrypt any data on the volume.
-
-Encryption keys can also be downloaded as a safety measure, to allow
-decryption on a different system in the event of failure, or to allow
-the locally stored key to be deleted for extra security. Encryption
-keys can also be optionally protected with a *passphrase* for
-additional security. The combination of encryption key location and
-whether a passphrase is used provide several different security
-scenarios:
+decrypt any data on the volume. 
+
+By default, %brand% stores encryption keys locally within its system
```

Comment:
```suggestion
%brand% stores encryption keys locally within the system
```

Reply:
It doesn't always store it (you can remove the local copy). Surely better to say "by default stored" in the first place, rather than somewhat mislead with information that has to later be qualified, when its that easy to give accurately in the first place?

Reply:
"The system" is good no question tho